scripts/install-mender.sh

```diff
@@ -266,12 +281,48 @@ do_install_commercial() {
     echo "  Success!"
 }
 
+do_setup_mender() {
+    # Return if mender-client was not installed
+    if [[ ! "$SELECTED_COMPONENTS" == *"mender-client"* ]]; then
+        return
+    fi
+
+    # Return if no setup options were passed
+    if [ -z "$MENDER_SETUP_ARGS" ]; then
+        return
+    fi
+
+    echo "  Setting up mender with options: $MENDER_SETUP_ARGS"
+    mender setup $MENDER_SETUP_ARGS
+    pidof systemd && systemctl restart mender-client
```

Comment:
Q:
What is the pidof for?
Isn't systemd always pid einz? :thinking:

Reply:
I added this as a simple way to check that `systemd` is running. Specifically, it is not running on tests so we skip calling `systemctl`

Reply:
Oh, I see, makes sense!
haha, did not strike me at all :see_no_evil: 
Still learning :seedling: